data/lua/helper.lua

```diff
@@ -301,7 +298,9 @@ function helper.shuffle( t, random_func)
 	random_func = random_func or wesnoth.random
 	-- since tables are passed by reference, this is an in-place shuffle
 	-- it uses the Fisher-Yates algorithm, also known as Knuth shuffle
-	assert( type( t ) == "table", string.format( "helper.shuffle expects a table as parameter, got %s instead", type( t ) ) )
+	assert(
+		type(t) == "table",
+		string.format( "helper.shuffle expects a table as parameter, got %s instead", type( t )))
```

Comment:
Inconsistent use of spaces.

Assuming that you want to keep using spaces within parentheses like the rest of this function, you also need to use them between the last closing parentheses:

```lua
string.format( "helper.shuffle expects a table as parameter, got %s instead", type( t ) ) )
```

Reply:
*sigh* just didn't want to either re-write the whole function-s spaces, and write code that I don't really like. OK, will go for uniformity and will add spaces.